### ISSUE TITLE
Turnover Time Defaults

### ISSUE THREAD

**Issue by DanielNScott** [EDmodel/ED2#61]
Check out the code below from ed_params.f90. The slow soil carbon decay rate is really high, and it seems plausible that people are running the model without nitrogen limitation OR a knowledge that their runs have nutty decay rates. Maybe it's time to change this, or at least make some note of it in the ED2IN?

```fortran
   !---------------------------------------------------------------------------------------!
   ! MLO.  After talking to Paul, it seems the decay rate for the slow carbon pool is      !
   !       artificially high for when nitrogen limitation is turned on.  If it is turned   !
   !       off, however, then the slow carbon will disappear very quickly.  I don't want   !
   !       to mess other people's results, so I will change the rate only when             !
   !       decomp_scheme is 2, and only when nitrogen limitation is off.  I think this     !
   !       should be applied to all schemes, but I will let the users of these schemes to  !
   !       decide.                                                                         !
   !---------------------------------------------------------------------------------------!
   select case (decomp_scheme)
   case (0,1)
      decay_rate_fsc  =  11.0 / yr_day    ! former K2
      decay_rate_stsc =   4.5 / yr_day    ! former K1
      decay_rate_ssc  = 100.2 / yr_day    ! former K3
   case (2)
      decay_rate_fsc  =  11.0 / yr_day    ! former K2
      decay_rate_stsc =   4.5 / yr_day    ! former K1
      select case (n_decomp_lim)
      case (0)
         decay_rate_ssc  =   0.2 / yr_day ! former K3
      case (1)
         decay_rate_ssc  = 100.2 / yr_day ! former K3
      end select
   end select
   !---------------------------------------------------------------------------------------!
```

**Comment by apourmok** [comment]
If I recall correctly, last year Jaclyn Matthes found the same problem and fixed it in her branch but not sure if she pushed that in to mainline.

**Comment by crollinson** [comment]
I was just looking at some of the outputs from our runs and our heterotrophic respiration is super high as well, so I just made a commit with Jackie's changes that Afshin was referring to.  From my understanding these are from a meta-analysis from Moyano et al 2012 in Biogeosciences.  You can scope out the changes here:  https://github.com/crollinson/ED2/commit/ab62e0964095b0943b249258aa75004716d39878

However, I think my problem may actually be linked to not having the system solved for steady-state correctly, but I'll leave the changes in place for now. I think the idea at the time was to allow this to be another option for decomp scheme and it was just never fully implemented.

**Comment by DanielNScott** [comment]
@apourmok it would seem she didn't, since the quoted code is in the mainline.

@crollinson Interesting... with too-high parameters the model can only develop respiratory CO2 at the rate of substrate input which, if the forest C pools are in steady state (@ harvard they're presumed to be, except the very old ones) is the rate it would be developing CO2 anyway. This would lead me to believe that if your resp is too high, and you aren't accumulating an unreasonable amount of fast soil C that you have too much input.

Thanks for the link to the different scheme, it does look like it would make sense potentially to add another case (but probably not to simply replace the default!) although we have so many options in the ED2IN it's already difficult to know what to make of many of them. I think an accounting of the various options and when/why they should be used or when/why they provide better results is in order for ED generally...

Anyway, I'm not actually sure all that's super relevant to this thread, or maybe I'm just missing it. The real question is, do we think as a community it's wise to have the default slow soil carbon turnover at 10 times the rate of fast soil carbon(!)? I suspect the answer is no.

**Comment by mpaiao** [comment]
@DanielNScott I agree with you, we should have reasonable default numbers for the decay rates for all cases.  For reference, below is the paper Paul suggested me to look at when I found this bug sometime in 2012.  I couldn't find any reason for the rate to be 100.2 (for example, check their Fig. 3).  It's probably just a typo that somehow has survived in the code since ED-1... 

B. M. Bolker, S. W. Pacala, and W. J. Parton. Linear analysis of soil decomposition: insights from the CENTURY model. Ecol. Appl., 8(2):425–439, May 1998. http://dx.doi.org/10.1890/1051-0761(1998)008[0425:LAOSDI]2.0.CO;2
